src/utils/utilsTests/searchUtils.test.ts

```diff
@@ -250,5 +250,20 @@ describe("searchUtils", () => {
         expect(heading).toEqual('Displaying 3 of 3 results for LCCN "1234"')
       })
     })
+    describe("browse result searches", () => {
+      it("returns heading with browseOptions when slug and browseType are provided", () => {
+        const heading = getSearchResultsHeading(
```

Comment:
is this a valid case? where there would be an advanced search query param along side a subject slug?